### ISSUE TITLE
Make testrunner script generic

### ISSUE THREAD

**Issue by manuel** [collectivemedia/celos#75]
The [manuel_sample_workflow](https://github.com/collectivemedia/celos/compare/manuel_sample_workflow) branch contains an initial implementation of a test running script.

It takes a [sample workflow](https://github.com/collectivemedia/celos/tree/manuel_sample_workflow/samples/file-copy), uploads it to the virtual cluster, steps the scheduler until no more jobs are running, and then compares the workflow's outputs against fixtures.

This should be generalized so arbitrary workflows can be tested:
- A workflow specifies its inputs and outputs under `src/test/celos-input` and `src/test/celos-output`.
- Running a script from the workflow directory performs all steps currently done by the `test.yaml` script, but in a generic way.

Basically, it should be possible to integration test a workflow by calling a single generic script from the workflow's repo / build process.
